api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/TxCurrCohortQueries.java

```diff
@@ -841,6 +790,30 @@ private CohortDefinition getPatientsWithMonthlyTypeOfDispensation() {
     cd.addValue(hivMetadata.getMonthlyConcept());
     return cd;
   }
+  
+  public SqlCohortDefinition getPatientsWithMonthlyTypeOfDispensation(List < EncounterType > encounterTypes) {
```

Comment:
If encounter types are always going to be:
```
    Arrays.asList(
            hivMetadata.getARVPharmaciaEncounterType(),
            hivMetadata.getAdultoSeguimentoEncounterType());
```
there is no reason to use parameters and a for loop, please inline them in the query and use String.format like all the other queries in the project.